### ISSUE TITLE
Bug in Docs

### ISSUE THREAD

**Issue by tulbureandreit** [ourownstory/neural_prophet#695]
In the docs for the Core Module Documentation https://neuralprophet.com/html/forecaster.html

In the parameter list for the neuralprophet forecaster, at some point there is a bug

We see:
```
(bool) (global_time_normalization) –

Specifies global time normalization

Options

        (default) True: only valid in case of global modeling local normalization

        False: set time data_params locally
```
Instead of : 
```
global_time_normalization (bool) –

Specifies global time normalization

Options

        (default) True: only valid in case of global modeling local normalization

        False: set time data_params locally
```

It took me 3 mins to figure out what`s wrong haha

Hope I was helpful

**Comment by tulbureandreit** [comment]
Another one in  https://neuralprophet.com/html/df_utils.html
```
neuralprophet.df_utils.double_crossvalidation_split_df(df, n_lags, n_forecasts, k, valid_pct, test_pct)

    Splits data in two sets of k folds for crossvalidation on validation and test data.

    Parameters

            (pd.DataFrame) (df) –

            (int) (k) –

            (int) –

            (int) –

            (float) (test_pct) –

            (float) –

    Returns

        elements same as [crossvalidation_split_df()](https://neuralprophet.com/html/df_utils.html#neuralprophet.df_utils.crossvalidation_split_df) returns
    Return type

        tuple of k tuples [(folds_val, folds_test), …]
```

**Comment by ourownstory** [comment]
@noxan Somehow I can't assign you, but do you mind looking into this and making sure it is fixed in the current docstring?

**Comment by ourownstory** [comment]
Thank you @tulbureandreit for raising this issue!

**Comment by LeonieFreisinger** [comment]
will have a look at it

**Comment by ourownstory** [comment]
seems to have been fixed in #782
